### PR TITLE
First Public Module

### PR DESCRIPTION
**Pull request by blaggacao**
Hi, this is the first public release of our l10n_co_geo module, which finally is stable and production proven.
I would kindly request, to put myself and @kurkop on the maintainers list for this repo, so that we can coordinate the localisation efforts. I think at this moment our legitimacy of doing so is without questioning and I would be happy to provide further information.

### CODE REVIEW

**Comment by blaggacao** [comment]
@dreispt @nhomar @pedrobaeza consider this WIP until everything gets green. I just copied the files into the directory and am yet to explore/familiarize with the l10n-oca repos structure. If you have general feedback, I'd be happy to discuss. 

We would like to publish more modules as we become sure that their design is production proven and undisputed. We would really appreciate to evaluate the possibility of becoming the maintainers here.

File: README.md [hunk 1]
```diff
@@ -3,11 +3,32 @@
...
+update del modulo para obtener la ultima version
+
+# Code Contributors
+- DevCO Colombia SAS (dar@devco.co)
+
+# Reviewers/Maintainers
+- DevCO Colombia SAS (dar@devco.co)
 
 [//]: # (addons)
+Available addons
```
Comment (pedrobaeza) [inline README.md]: This is done automatically, so no need to change it.

File: README.md [hunk 2]
```diff
@@ -3,11 +3,32 @@
...
+
+------------------
+Esto es la localisacion oficial comunitaria de Colombia. Encontraras modulos probados en 
+produccion que estan en un estado estable. Si hay algun upgrade, requerimos de los 
+contribuidores de proveer scripts de migracion basandose en openupgradelib. Migraciones especiales
+van a tener instrucciones por separada. En el caso comun, sin embargo, sera suficiente un simple
+update del modulo para obtener la ultima version
+
+# Code Contributors
+- DevCO Colombia SAS (dar@devco.co)
```
Comment (pedrobaeza) [inline README.md]: Main README.md doesn't contain contributors. Each module readme does.

File: l10n_co_geo/__init__.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+###############################################################################
```
Comment (pedrobaeza) [inline l10n_co_geo/__init__.py]: Use headers like https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: l10n_co_geo/__init__.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        res_country_state_city,
+        res_country_state,
+        res_country,
+        res_partner,
+        res_company
+)
+
+
+def _fill_city_id(cr, registry):
+    ''' This is a utiliy function to help fill res_partner city_id, state and country with Bogota, Dc, Colombia.
```
Comment (pedrobaeza) [inline l10n_co_geo/__init__.py]: Double quote for docstring according PEP257

Comment (pedrobaeza) [reply l10n_co_geo/__init__.py]: This hook method should be in a file hooks.py

File: l10n_co_geo/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,120 @@
+# -*- encoding: utf-8 -*-
```
Comment (pedrobaeza) [inline l10n_co_geo/__openerp__.py]: s/encoding/coding

File: l10n_co_geo/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+#
+#    Odoo, Open Source Management Solution
+#    This module copyright (C) 2015 DevCO Colombia SAS
+#    (http://devco.co).
+#
+##############################################################################
+
+{
+    'name': 'Colombian Geodata',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline l10n_co_geo/__openerp__.py]: 10.0.1.0.0

File: l10n_co_geo/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    'author': 'DevCO Colombia SAS',
+    'maintainer': 'DevCO Colombia',
+    'website': 'http://devco.co',
+    'license': 'AGPL',
+    'category': 'Localization',
+    'summary': 'such as cities, departments and some improvements on displaying them.',
+    'description': """
+
+Colombian Geodata
```
Comment (pedrobaeza) [inline l10n_co_geo/__openerp__.py]: Move this to a README.rst file and follow latest template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: l10n_co_geo/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+* Juan Pablo Arias <jpa@devco.co>
+
+Maintainer
+----------
+
+.. image:: https://www.devco.co/logo.png
+   :alt: DevCO Colombia SAS
+   :target: http://devco.co
+
+This module is maintained by DevCO Colombia SAS.
```
Comment (pedrobaeza) [inline l10n_co_geo/__openerp__.py]: Maintainer is OCA, not DevCo

File: l10n_co_geo/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+DevCO Colombia is a Colombian Odoo Specialist whose mission is to provide the most comprehensive colombian localisation and a broad range of next generation hosting and implementation services.
+
+For further information, please visit http://devco.co.
+""",
+
+    # any module necessary for this one to work correctly
+    'depends': [
+        'base',
+    ],
+    'external_dependencies': {
```
Comment (pedrobaeza) [inline l10n_co_geo/__openerp__.py]: Remove empty keys and comments

File: l10n_co_geo/models/res_company.py [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+###############################################################################
+
+from openerp import models, fields, api
+
+
+class ResCompany(models.Model):
+    _name = 'res.company'
+    _inherit = 'res.company'
+
+    country_id = fields.Many2one(
```
Comment (pedrobaeza) [inline l10n_co_geo/models/res_company.py:11]: You shouldn't convert this to a related field. Take into account that you can mix companies for several countries, not only Columbian ones.

File: l10n_co_geo/models/res_company.py [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        invisible=True
+    )
+
+    state_id = fields.Many2one(
+        'res.country.state',
+        related='city_id.state_id',
+        readonly=True,
+    )
+
+    city_id = fields.Many2one(
```
Comment (pedrobaeza) [inline l10n_co_geo/models/res_company.py:35]: There's already a module in OCA that adds a city/zip structure: https://github.com/OCA/partner-contact/pull/317. Please use it instead of reinventing the wheel.

File: l10n_co_geo/models/res_country_state.py [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+###############################################################################
+#    License, author and contributors information in:                         #
+#    __openerp__.py file at the root folder of this module.                   #
+###############################################################################
+
+from openerp import models, fields
+
+
+class ResState(models.Model):
+    _name = 'res.country.state'
```
Comment (pedrobaeza) [inline l10n_co_geo/models/res_country_state.py]: No need to put the name if it's the same when you're inheriting.

File: l10n_co_geo/models/res_country_state.py [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+    _inherit = 'res.country.state'
+
+    iso = fields.Char(
```
Comment (pedrobaeza) [inline l10n_co_geo/models/res_country_state.py:11]: This is already done in this module: https://github.com/OCA/community-data-files/tree/8.0/base_iso3166, so no need to duplicate. The migration to 10.0 is direct.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
David, about your proposal to be PSC members for Columbia, you need to write to contribute@odoo-community.org for requesting that.

File: README.md (hunk 2 above)
Comment (blaggacao) [reply README.md]: OK. Thanks!

File: README.md (hunk 1 above)
Comment (blaggacao) [reply README.md]: OK. cool!

File: l10n_co_geo/__init__.py (hunk 3 above)
Comment (blaggacao) [reply l10n_co_geo/__init__.py]: Alright, need to change my snippets then... OCA ST3 snippets one day? 😄

File: l10n_co_geo/__init__.py (hunk 4 above)
Comment (blaggacao) [reply l10n_co_geo/__init__.py]: Good idea! Two questions: 1. Is this OCA standard or even Odoo standard? 2. Only for init hooks or is the idea to have any kind of hook, also in normal code, clustered in a special file. ... I definitely see the use for this practice.

File: l10n_co_geo/__openerp__.py (hunk 7 above)
Comment (blaggacao) [reply l10n_co_geo/__openerp__.py]: OK, thanks! Need to look at this template folder and find a way to make it my standard template... I also want to include the oca-pylint-checks into my ST3 but it's not obvious yet, need to see, because those are great tools..

File: l10n_co_geo/__openerp__.py (hunk 8 above)
Comment (blaggacao) [reply l10n_co_geo/__openerp__.py]: Alright, I thought that one. Devco would figure as contributor on the module, only.

File: l10n_co_geo/__openerp__.py (hunk 9 above)
Comment (blaggacao) [reply l10n_co_geo/__openerp__.py]: alright.

File: l10n_co_geo/models/res_company.py (hunk 10 above)
Comment (blaggacao) [reply l10n_co_geo/models/res_company.py:11]: That's a design decision, if you are in Colombia, you always **need** the city, if you are dealing with an external client, you always **know** the city. Both times, they reliably give you the country. Because in colombia, the city is obligatory, we have the city `required=True`, so in the end it gives almost the same. Would you agree? Or do you have a better alternative to solve this? Beware of that city is a model with metadata of it's own.

File: l10n_co_geo/models/res_company.py (hunk 11 above)
Comment (blaggacao) [reply l10n_co_geo/models/res_company.py:35]: That's a nice solution, as our private code definitely depends on this functionality, we would come into license conflict, so internally we cannot use the linked module. This published code is relicensed under a private license for internal use, but we cannot do this with the other module. I'm thinking of a technical structure we can incorporate this module here, without breaking our licensing structure. Do you have any ideas?

File: l10n_co_geo/models/res_country_state.py (hunk 13 above)
Comment (blaggacao) [reply l10n_co_geo/models/res_country_state.py:11]: Nice, however, same as above, due to licensing concerns. Any advice?

**Comment by blaggacao** [comment]
@pedrobaeza Thanks for the review, it's a great learning experience!

All done, but pending:
https://github.com/OCA/l10n-colombia/pull/4#discussion_r85482879 (Design Question)
https://github.com/OCA/l10n-colombia/pull/4#discussion_r85483010 (License Question)
https://github.com/OCA/l10n-colombia/pull/4#discussion_r85483149 (Licesne Question)

File: l10n_co_geo/README.rst [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+* Juan pablo Arias <jpa@devco.co>
+
+Sponsors
+---------
+
+.. image:: https://www.devco.co/logo.png
+   :alt: DevCO Colombia SAS
+   :target: http://devco.co
+
+This module is sponsored by DevCO Colombia SAS.
```
Comment (pedrobaeza) [inline l10n_co_geo/README.rst]: The module can't carry this commercial information. You have the author and the website for that. This is because you have started the module, but once in OCA, a lot of other people will start to contribute to it, and a brand war cannot start. That's why the decision is to put no brand at all.

File: l10n_co_geo/data/res.country.csv [hunk 15]
```diff
@@ -0,0 +1,254 @@
+"id","code_dian","name"
```
Comment (pedrobaeza) [inline l10n_co_geo/data/res.country.csv]: Why do you load again countries? They are already in the base.

File: l10n_co_geo/data/res_bank.xml [hunk 16]
```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <!-- <record model="res.bank" id="bank0">
```
Comment (pedrobaeza) [inline l10n_co_geo/data/res_bank.xml]: Remove commented entries.

File: l10n_co_geo/data/res_bank.xml [hunk 17]
```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pedrobaeza) [inline l10n_co_geo/data/res_bank.xml]: We put this file in other module called `l10n_es_partner` to associate certain logic to deduce the bank from the partner bank account.

File: l10n_co_geo/data/res_config_co.xml [hunk 18]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <!-- Install es_CO language -->
+    <record model="base.language.install" id="install_co">
```
Comment (pedrobaeza) [inline l10n_co_geo/data/res_config_co.xml]: You mustn't install language this way. Anyway, you must split UI from data. You should be able to install this module without being forced to install Colombian language, so please don't do that. The one that wants to have the language, can use the regular "Load translation" option for doing that.

File: l10n_co_geo/data/res_country_state.xml [hunk 19]
```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pedrobaeza) [inline l10n_co_geo/data/res_country_state.xml:1]: base_location_geonames_import includes this information directly.

File: l10n_co_geo/models/res_company.py (hunk 11 above)
Comment (pedrobaeza) [reply l10n_co_geo/models/res_company.py:35]: You have the same problem if you are using this module, as the chosen license is AGPL-3. But are you sure you need to depend on that code. I think you can split the functionality for not depending on this, but I don't know the use case.

File: l10n_co_geo/README.rst (hunk 14 above)
Comment (blaggacao) [reply l10n_co_geo/README.rst]: ok, very understandable. Sorry for that, I should have known.

File: l10n_co_geo/data/res.country.csv (hunk 15 above)
Comment (blaggacao) [reply l10n_co_geo/data/res.country.csv]: It's because of the `code_dian` column, which is needed. Is the way we do it valid? I don't think that this is any standard code, but rather DIAN herself invented it (Dirección de Impuestos y Aduanas Nacionales)

File: l10n_co_geo/models/res_company.py (hunk 10 above)
Comment (pedrobaeza) [reply l10n_co_geo/models/res_company.py:11]: But you have to think always globally: if you don't have city entry for other countries, you won't be able to put the country. Put an onchange, or if you wan to hard-relate both when a city entry is filled, then overwrite write and create methods for changing country_id field according that, but this second one is no usually needed, and more now with v10 version that is very simple to call new onchange methods transparently.

File: l10n_co_geo/models/res_country_state.py (hunk 13 above)
Comment (pedrobaeza) [reply l10n_co_geo/models/res_country_state.py:11]: The same: I don't know for what do you use this, but normally it's possible to split things, or just don't depend on the module and detect dynamically if the fields are present.

File: l10n_co_geo/data/res_bank.xml (hunk 16 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_bank.xml]: I left them because I'm not sure if they are actually relevant banks or not. I would like to leave them there to start discussion among contributors and not loose the base set of information. Is this goal achievable?

I mean they all have a colombian banking license, so theoretically, somebody could have an account with them, but sorted them out based on my own judgment, because they might be close to irrelevant.

File: l10n_co_geo/data/res.country.csv (hunk 15 above)
Comment (pedrobaeza) [reply l10n_co_geo/data/res.country.csv]: OK, I see. The problem with CSV is that you need the name column, so you are overwriting it and you need to maintain it (with translations). This won't be needed in case you put the other module (or is a new code apart from the ISO?). Anyway, if needed, I recommend you 2 things:
- Extract it in a separate module for being more modular and don't depend on such big modules like this one.
- Load it with a hook, having a Python dictionary (code as data) with pairs of XML-IDs as keys and DIAN codes as values. Then, loop over the dictionary, making something similar to (pseudo-code):

```
for xml_id, code in dictionary:
   env.ref(xml_id).dian = code
```

File: l10n_co_geo/data/res_bank.xml (hunk 17 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_bank.xml]: Is this an imperative or an optional recommendation? We have a module `l10n_co_terceros` which is loading additional data, also on banks. Do you mean it would be wise to split out the loading of banks out of this module? Let's say into a `l10n_co_bank_data`? I can see an advantage of splitting things into smaller units to benefit more focused discussion and collaboration. Asking for clarification...

File: l10n_co_geo/data/res_bank.xml (hunk 16 above)
Comment (pedrobaeza) [reply l10n_co_geo/data/res_bank.xml]: Well, if they are actually valid, I would let them loaded. If not, then include a comment saying so for anyone to come to the code knows your reasons.

File: l10n_co_geo/data/res_bank.xml (hunk 17 above)
Comment (pedrobaeza) [reply l10n_co_geo/data/res_bank.xml]: Not imperative, but a way to keep together some functionality as modular. For making people that don't use banks (yes, they exist), to not need to load them in the base module.

File: l10n_co_geo/data/res_config_co.xml (hunk 18 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_config_co.xml]: @kurkop entiendes la retroalimentación de Pedro? Esto significa que no podemos utilizar este modulo clandestinamente en los desplieges para este fin.
@pedrobaeza ok, this needs some refactoring of our deployment strategy, but you're absolutely right.

File: l10n_co_geo/data/res_country_state.xml (hunk 19 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_country_state.xml:1]: Nice idea, but as far as I can see, ther is no CO.zip (http://download.geonames.org/export/zip/) Or is it inlcuded in the allCountries.zip? In any case, we still would need to include the DIAN code which is not any of ISO or ZIP. Any ideas of how we could integrate this? Including feed information back into geonames.org? Especially, if we could build a base for Colombian ZIP code this would be cool, as it is barely used yet.

**Comment by blaggacao** [comment]
Round 2 remain pending:

https://github.com/OCA/l10n-colombia/pull/4#pullrequestreview-6343446 (Special permit to leave them there?)
https://github.com/OCA/l10n-colombia/pull/4#pullrequestreview-6343450 (Establish right degree of granularity and refactor bank data)
https://github.com/OCA/l10n-colombia/pull/4#pullrequestreview-6343469 (Outfactor deployment code @kurkop)

File: l10n_co_geo/data/res_country_state.xml (hunk 19 above)
Comment (pedrobaeza) [reply l10n_co_geo/data/res_country_state.xml:1]: OK, I see that this country is not yet considered to be packed as ZIP. This is because low demand. I don't know the mechanisms for uploading or demanding this pack. Maybe you can ask in geonames itself. Then continue this way until there's a better solution.

**Comment by blaggacao** [comment]
Let's stop for a moment, I'm working on get travis checks green, now. THen we can proceed to resolve the open points.

@pedrobaeza Meanwhile we can probably find answers to the open points?

File: l10n_co_geo/data/res_bank.xml (hunk 16 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_bank.xml]: I think, l'll opt for the second, who has a bank account with Allianz Seguros de Vida SA? 😄 I think this can be marked as solved. Thanks!

File: l10n_co_geo/models/res_company.py (hunk 11 above)
Comment (blaggacao) [reply l10n_co_geo/models/res_company.py:35]: As the author I can relicense it in it's current state, nobody can force me to only license it under AGPL only, so juristically speaking, we can cleanly separate concerns. Obviously, once someone else gains co-authorship we would need to maintain a fork and cannot use adaptations. Now I think strictly speaking we do not depend on the code, rather on the data the code makes possible, with checks like "if this client is habitant of this city, then do something", but I think we have a problem as well, because we would need to release the data under AGPL, too, because obviously it's dependent on the code, and in consequence, everything that depends on the data needs to be made available under AGPL too. This is very problematic as we need to protect our "inventions" at least for some time...

In conclusion, if we refactor the code to use the suggested base modules, which technically speaking is a very good idea, then we would already right from the beginning need to maintain a fork in order to keep legal concerns separated, AND furthermore, clients who once might come to us for support would run under a different code base. Now that's not something easy to resolve, I guess, not even to speak about its economic viability...

File: l10n_co_geo/data/res_bank.xml (hunk 17 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_bank.xml]: You convinced me that putting it in a separate module is probably better. I'll put it `l10n_co_partner_bank_data` that makes attribution clear, and separates concerns.

File: l10n_co_geo/data/res_country_state.xml (hunk 19 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_country_state.xml:1]: I investigated on one of geonames sources (http://geonames.nga.mil/namesgaz/) which is also very interesting, I'll get in contact with them and let's see what we can do. This definitely can be a good thing.

File: l10n_co_geo/models/res_company.py (hunk 11 above)
Comment (blaggacao) [reply l10n_co_geo/models/res_company.py:35]: @pedrobaeza I changed the licensing to LGPL altogether, your comment, although not juridically correct, made me fear sueing from unfriendly competitors against our online offer, who might have the same interpretation... Better not put ourselves at risk.

File: l10n_co_geo/data/res_country_state.xml (hunk 19 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_country_state.xml:1]: I see they have an ambassador for Colombia already, the problem is I guess, that ZIP codes are not openly licensed. Stupid thing, but things are as they are. (2 years ago was the last time, I checked).

**Comment by blaggacao** [comment]
The strange menuitem module is due to this unlucky decision in 10.0: https://github.com/odoo/odoo/blob/10.0/addons/sales_team/views/crm_team_views.xml#L20

File: l10n_co_geo/data/res_country_state.xml (hunk 19 above)
Comment (blaggacao) [reply l10n_co_geo/data/res_country_state.xml:1]: I just saw that those get increasingly merged into `upstream/10.0`. That would be the most elegant of all solutions, i guess...

File: l10n_co_geo/hooks.py [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+
+def _fill_city_id(cr, registry):
+    """
+    This is a utiliy function to help fill res_partner city_id,
+    state and country with Bogota, Dc, Colombia.
+    """
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    Partner = env['res.partner']
+    all = ['|', ('active', '=', False), ('active', '=', True)]
+    Partners = Partner.search(all + [('city_id', '!=', None)])
```
Comment (blaggacao) [inline l10n_co_geo/hooks.py:16]: @kurkop, por que es este leaf? `[('city_id', '!=', None)]` parece fuertemente ser erroneo a la primera vista. Viene de esta forma de la versión inicial.

**Review by blaggacao — COMMENTED** [review]
@kurkop can you have a look at this review?

**Comment by blaggacao** [comment]
@pedrobaeza Everything is green now, I think we can continue and sort out the open points. Maybe you can release the remaining two request for changes with the following proposal:
- We'll work to include the res_country_state into upstream?
- We'll propose a license change for most basic OCA modules in the `partner-contact` repository, in order to try to enhance their (legal) usability OR we might push to include cities already into upstream in order to deprecate `better.zip` alltogether. I guess upstream, the argument is the sheer size of having lot's of cities incorporated, but who would object having a base model...

https://github.com/OCA/partner-contact/issues/321

Meanwhile, I'm awaiting further input...

**Comment by blaggacao** [comment]
Here we go, with the states: https://github.com/odoo/odoo/pull/14067

**Comment by pedrobaeza** [comment]
David, ninguno de los contribuidores va a querer cambiar a LGPL. Tal vez deberíais plantearos cambiar vosotros a AGPL. Por mi experiencia, una inversión no se protege privatizando el código, si no todo lo contrario: se consigue que haya muchas versiones alternativas y ninguna se consolida, entrando en una lucha de localizaciones en la que ninguno sale beneficiado. En cambio, aquel que es el primero que libera totalmente (y en OCA), consigue ser "LA LOCALIZACIÓN", tener esa imagen, ese know-how, y por tanto, ser la referencia a la que contactan los leads.

**Comment by blaggacao** [comment]
Listo, entiendo eso. No es bloqueante para mi. Nosotros realmente no podemos irnos a AGPL porque estamos planeando acceder a fuentes no-AGPL en el futuro próximo que podría resultar en un conflicto no fácil de resolver así:
Fuente A: AGPL
Fuente B: Licencia Privada, que licenciamos por cuenta de nosotros
Código de nosotros "C" que depende tanto de A como de B. No sabria decir como se puede resolver, pero segun mi entendimiento, simplemente no se puede. Ahora bien, si datos tan centrales introducen AGPL, ahora bien estamos perdidos.

Digamos, si obviamente, puede haber modulos AGPL, pero es un gran riesgo de tener modulos claves en AGPL creo que un approach mixto, pragmático y no-ideológico es la solución.

File: l10n_co_geo/data/res_config_co.xml [hunk 21]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <!-- until included in https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/res/res.lang.csv -->
+  <!-- https://github.com/odoo/odoo/pull/14065 -->
+  <record id="base.lang_es_CO" model="res.lang">
+    <field name="date_format">%d-%m-%Y</field>
```
Comment (kurkop) [inline l10n_co_geo/data/res_config_co.xml:6]: @blaggacao acá antes estaba: 

```
<record id="base.lang_es_CO" model="res.lang">
    <field name="grouping">[3,0]</field>
    <field name="date_format">%d-%m-%Y</field>
    <field name="decimal_point">,</field>
    <field name="thousands_sep">.</field>
</record>
```

¿Por qué lo quitó?
Hace unos meses hice un PR a Odoo con esa configuración pero creo que no lo han aceptado.

Comment (blaggacao) [reply l10n_co_geo/data/res_config_co.xml:6]: Solo deje la diferencia. Si veas el upstream, el resto ya esta. No veo contundente sobrescribir ciegamente, aunque los valores esten correctos upstream.
